rdflib/parser.py

```diff
@@ -214,3 +219,75 @@ def create_input_source(source=None, publicID=None,
         elif input_source.getPublicId() is None:
             input_source.setPublicId(absolute_location or "")
         return input_source
+
+SUFFIX_FORMAT_MAP = {
```

Comment:
This is just a copy of the code in https://github.com/RDFLib/rdflib/blob/master/rdflib/util.py#L345? Why not import it instead?

Reply:
I tried importing, but I was not able to. It led to many test cases failing.